reviews/forms.py

```diff
@@ -102,7 +102,7 @@ def __init__(self, *args, **kwargs):
         allow_long_route_continuation = kwargs.pop('allow_long_route_continuation', False)
         super(ReviewCloseForm, self).__init__(*args, **kwargs)
         if not allow_long_route_continuation:
-            self.fields['continuation'].choices = [x for x in Review.CONTINUATIONS if x[0] != Review.LONG_ROUTE]
+            self.fields['continuation'].choices = [x for x in Review.Continuations.choices if x != Review.Continuations.LONG_ROUTE]
```

Comment:
This does not work?:

```
>>> [x for x in Review.Continuations.choices if x != Review.Continuations.LONG_ROUTE]
[(0, 'Goedkeuring door FETC-GW'), (1, 'Revisie noodzakelijk'), (2, 'Afwijzing door FETC-GW'), (3, 'Open review met lange (4-weken) route'), (4, 'Laat opnieuw beoordelen door METC'), (5, 'Positief advies van FETC-GW, post-hoc'), (6, 'Negatief advies van FETC-GW, post-hoc'), (7, 'Niet verder in behandeling genomen')]
```
I think it should be this:

```suggestion
            self.fields['continuation'].choices = [x for x in Review.Continuations.choices if x[0] != Review.Continuations.LONG_ROUTE]
```


As for a better solution: there isn't any really that's clearly better. One alternative is a filter:
```python
filter(lambda x: x[0] != Review.Continuations.LONG_ROUTE, Review.Continuations.choices)
```
But that's not really that much better.

You could also create a method in the new class to do the list comprehension there, but for this one usage that does not make much sense

Reply:
I'm sorry. I just assumed ... Too bad that does not work. I just used your first suggestion.